addons/binding/org.openhab.binding.miio/doc/vacuum.md

```diff
@@ -0,0 +1,77 @@
+# Xiaomi Robot Vacuum Binding
```

Comment:
Please consider adding this information in the main README. Multiple md files for one binding are unusual and not explicitly supported by docs.openhab.org.